website/content/partials/cli/token/flags/format.mdx

```diff
@@ -0,0 +1,18 @@
+<a id="global-format" />
```

Comment:
`format` is not a command-specific flag, it's a global flag that will be include automatically if you use `@include 'cli/standard-settings/all-standard-flags.mdx'`